google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryStage.java

```diff
@@ -428,21 +559,21 @@ ExplainQueryStage toPb() {
 
   static QueryStage fromPb(com.google.api.services.bigquery.model.ExplainQueryStage stagePb) {
     Builder builder = new QueryStage.Builder();
```

Comment:
What is the problem with this? This is internal code that accesses directly the constructor rather than going through the public builder method, I don't see an issue with that.